version/version.go

```diff
@@ -1,5 +1,5 @@
 package version
 
 var (
-	Version = "0.0.1"
+	Version = "0.1.0"
```

Comment:
When does this operator get its major release?

Reply:
Not sure :thinking: maybe once we're sure it's ready, and have moved the CRD versions to a v1 version (currently v1alpha1, like in our other operators). I think this will become clear once it's being used in a few places.